### PR TITLE
feat: allow usage outside Vue components

### PR DESCRIPTION
**Pull request by dobromir-hristov**
## Summary

Allows using Vuelidate outside of Vue Components. No breaking changes or API changes.

## Details

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] It's submitted to the `dev` branch for v1.x (or to a previous version branch), _not_ the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] All tests are passing: https://github.com/vuelidate/vuelidate/blob/master/.github/CONTRIBUTING.md#development-setup
- [x] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

File: packages/vuelidate/src/index.js [hunk 1]
```diff
@@ -108,7 +110,13 @@ export function useVuelidate (validations, state, globalConfig = {}) {
...
   const resultsCache = new ResultsStorage()
 
-  const { childResults, sendValidationResultsToParent, removeValidationResultsFromParent } = nestedValidations({ $scope, $stopPropagation })
+  const {
+    childResults,
+    sendValidationResultsToParent,
+    removeValidationResultsFromParent
+  } = instance
+    ? nestedValidations({ $scope })
+    : { childResults: ref('') }
```
Comment (shentao) [inline packages/vuelidate/src/index.js:119]: Should this be a `ref('')`?

**Review by shentao — APPROVED** [review]
Looks good! Just left one question.

File: packages/vuelidate/src/index.js (hunk 1 above)
Comment (dobromir-hristov) [reply packages/vuelidate/src/index.js:119]: It's a ref with an array by default. We just need something to fallback to. I guess we can do `ref([])`?

Comment (shentao) [reply packages/vuelidate/src/index.js:119]: Ah, then yeah I think keeping the types unified might be a good move.

**Pull request merged**